Step5.md

```diff
@@ -219,12 +219,43 @@ RUN printf "deb http://ftp.us.debian.org/debian jessie main\n" >> /etc/apt/sourc
 
 #### Building your container and application
 
+Let's package our application with only the coreclr so that the binary is nice and compact.
+
+```
+dnu publish --framework dnxcore50 --out publishedoutput
+cd publishedoutput
+```
+
+Then you want to teach Docker how to package an image containing your application,
+and how to execute the image by crafting a `Dockerfile`.
+
+```
+FROM microsoft/aspnet:1.0.0-rc1-final-coreclr
+COPY . /app/
+WORKDIR /app/approot
+EXPOSE 5000/tcp
+ENTRYPOINT ["./web", "--server.urls", "http://0.0.0.0:5000"]
```

Comment:
@teyc I'll take another look at this once I have landed the "chapter 5" changes from @hgaard as he's completely rewriting chapter 5 and is adding a new "docker deep dive" chapter 6a/6b. We'll make sure it works like the above once we do that.